x-pack/plugin/src/test/resources/rest-api-spec/api/ccr.delete_auto_follow_pattern.json

```diff
@@ -1,6 +1,7 @@
 {
   "ccr.delete_auto_follow_pattern": {
     "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-delete-auto-follow-pattern.html",
+    "stability" : "beta",
```

Comment:
I think all CCR APIs are GA https://github.com/elastic/elasticsearch/pull/39722

Reply:
ta, this PR preceeded that one. Updated now.